### PR TITLE
v1

### PR DESCRIPTION
**Pull request by Itchimonji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,22 +16,29 @@ $ nx serve
...
+- Read the data from the JSON file `apps/challenge/src/assets/time-series-data.json` and display it in the chart of component with the selector `time-series` dynamically.
+- What is wrong with the time stamp? Correct it dynamically in code and not in the raw data!
 
 ### Task 3
 
-Extend the application so that the time series data could also come from a REST API or other endpoints. The user should be able to change its source at runtime
+- Extend the application so that the time series data could also come from a REST API or other endpoints (Look a and b below). The user should be able to change its source at runtime. Attempts to ensure a high level of reusability and that other possible data sources can be easily integrated without having to make changes in many places in the code.
+- (a) The first additional data source is located in `apps/challenge/src/app/shared/services/mock.service.ts`. You can use the service globally.
+- (b) The second additional data source is located in `wird im nächsten commit eingefügt`
```
Comment (Taldorr) [inline README.md:31]: ?

**Review by Taldorr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Itchimonji) [reply README.md:31]: I need upload the data, so I can link it afterwards

Comment (Itchimonji) [reply README.md:31]: And "?" are not really helpful in PRs, because I could answer with "?", too ;)

Comment (Taldorr) [reply README.md:31]: I was just wondering about the `wird im nächsten commit eingefügt` :D

**Pull request merged**